pyracf/common/security_admin.py

```diff
@@ -84,6 +91,20 @@ def __overwrite_field_data(self, field_data: dict):
         """Overwrite field data in valid segment traits dictionary"""
         self._valid_segment_traits = field_data
 
+    def __add_additional_secret_traits(self, additional_secret_traits: list):
+        """Add additional fields to be redacted in logger output"""
```

Comment:
```python
        """Add additional fields to be redacted in logger output."""
```

Reply:
That was a good catch and a broader issue. Caught and fixed in many other locations as well.